Gopkg.lock

```diff
@@ -298,8 +298,7 @@
 [[projects]]
   name = "github.com/satori/go.uuid"
   packages = ["."]
-  revision = "f58768cc1a7a7e77a3bd49e98cdd21419399b6a3"
-  version = "v1.2.0"
+  revision = "master"
```

Comment:
Using `master` is bit risky. I would suggest to use the latest commit hash here.